.travis.yml

```diff
@@ -41,25 +41,19 @@ matrix:
         #- os: linux
         #  env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.13 SETUP_CMD='test'
 
-        # PEP8 check with pycodestyle (only once, i.e. "os: linux")
+        # PEP8 check with flake8 (only once, i.e. "os: linux")
         - os: linux
-          env: PYTHON_VERSION=3.5 MAIN_CMD='pycodestyle --count'
-               SETUP_CMD='jwst' TEST_CMD='pycodestyle --version'
+          env: PYTHON_VERSION=3.5 MAIN_CMD='flake8 --count'
+               SETUP_CMD='jwst' TEST_CMD='flake8 --version'
                CONDA_DEPENDENCIES=$CONDA_JWST_DEPENDENCIES
 
         # build sphinx documentation with warnings
         - os: linux
           env: PYTHON_VERSION=3.5
-               SETUP_CMD='build_sphinx'
+               SETUP_CMD='build_docs'
                CONDA_DEPENDENCIES=$CONDA_JWST_DEPENDENCIES
                PIP_DEPENDENCIES='sphinx_rtd_theme sphinx-automodapi'
 
-    allow_failures:
```

Comment:
Does this mean that PEP8 failures will no longer be classified as allowed and hence Travis will report that all tests have failed?

Reply:
Yes, unless it is one of the many ignored codes as stated in `setup.cfg` under `[flake8]` section. This is for the best because in my recent PEP8 campaign over at Ginga, I found several actual bugs hiding in the `flake8` warnings (e.g., undefined variables).

Reply:
By the way, there is quite a lot of undefined variables...